### ISSUE TITLE
Have you seen this message? Ran out of customers in DB?

### ISSUE THREAD

**Issue by patadeloso** [magento/magento-performance-toolkit#5]
You have to increase customers qty for running 'Customer Checkout' thread.
using user count set to 900

We used the small store profile. I think the test script ran out of users. Will try again with med/lg profile.

**Comment by peer1-fdirocco** [comment]
The small store profile only generates 20 users. so you either need to reduce the user count or increase the number of users you generate with your profile. Medium only generates 200 customers and large generates 2000 customers. If you want to hit 900 users you will need at least that many user fixtures.

you could simply change
```
<customers>20</customers> <!-- Number of customers to generate -->
```
to 
```
<customers>900</customers> <!-- Number of customers to generate -->
```
in small.xml and regenerate you profile in a fresh DB

**Comment by patadeloso** [comment]
Yeah, I'm trying to do a 2M SKU, 10K category, 50K user one right now ... they need to redo the gen script so its more memory efficient.
